lib/src/main/java/net/dean/jraw/models/SimpleFlairInfo.java

```diff
@@ -7,29 +7,48 @@
 import net.dean.jraw.databind.RedditModel;
 import org.jetbrains.annotations.Nullable;
 
+import java.util.List;
+
+/**
+ * SimpleFlairInfo contains basic information about user's flair
```

Comment:
120 line length please

Reply:
Fixed in https://github.com/mattbdean/JRAW/pull/207/commits/23681210a97da825e667139552bb19b2baa1f381